docs/DeveloperGuide.md

```diff
@@ -545,15 +545,16 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an empty folder.
```

Comment:
should we change the numbering to 1a, 1b, etc.?

Reply:
We could, but the formatting will look like this instead. 

<img width="744" alt="image" src="https://user-images.githubusercontent.com/84223259/200110284-b191e0aa-2899-4ec2-b2fe-ddc4ef1191be.png">

For reference, the current formatting is as such
<img width="701" alt="image" src="https://user-images.githubusercontent.com/84223259/200110295-c6ff7a7a-a7d5-4639-b6f5-0ae7d8bbaadc.png">

Alternatively, we can write CSS to account or that, or just ignore it. Thoughts? @ichigh0st

Reply:
ok then nvm css seems like a lot of effort

Reply:
The CSS will look something like the following, not too difficult if its what the preferred behaviour. 😄 

```html
<div style="display: flex; gap: 4px">
<span>1b.</span>
Re-launch the app by double-clicking the jar file.<br>
Expected: The most recent window size and location is retained.
</div>